Makefile

```diff
@@ -49,27 +59,28 @@ all: check \
 	build/packages.json \
 	build/test.html \
 	build/webworker.js \
-	build/webworker_dev.js
+	build/webworker_dev.js \
+
 	echo -e "\nSUCCESS!"
 
 
 build/pyodide.asm.js: \
-		src/core/error_handling.o \
-		src/core/hiwire.o \
-		src/core/js2python.o \
-		src/core/jsproxy.o \
-		src/core/keyboard_interrupt.o \
-		src/core/main.o  \
-		src/core/pyproxy.o \
-		src/core/python2js_buffer.o \
-		src/core/python2js.o \
-		src/core/runpython.o \
-		src/pystone.py \
-		src/_testcapi.py \
-		src/webbrowser.py \
-		\
-		$(wildcard src/pyodide-py/pyodide/*.py) \
-		$(CPYTHONLIB)
+	src/core/error_handling.o \
+	src/core/hiwire.o \
+	src/core/js2python.o \
+	src/core/jsproxy.o \
+	src/core/keyboard_interrupt.o \
+	src/core/main.o  \
+	src/core/pyproxy.o \
+	src/core/python2js_buffer.o \
+	src/core/python2js.o \
+	src/core/runpython.o \
+	src/pystone.py \
+	src/_testcapi.py \
+	src/webbrowser.py \
+	$(wildcard src/pyodide-py/pyodide/*.py) \
+	$(CPYTHONLIB) \
+
```

Comment:
I agree we should be more consistent in the makefile. I have mixed feeling on using an extra indentation vs a newline. WDYT @dalcde ?

Reply:
the double indentation you can see in the current master came in from the GitHub web-based editor when I solved a merge conflict some days ago. I don't know why it did that.